spec/syntax.md

```diff
@@ -691,8 +693,7 @@ A _literal_ can appear
 as a _key_ value,
 as the _operand_ of a _literal-expression_,
 or in the value of an _option_.
-A _literal_ MAY include any Unicode code point
-except for U+0000 NULL or the surrogate code points U+D800 through U+DFFF.
+A _literal_ MAY include any Unicode code point except for U+0000 NULL.
```

Comment:
If we add the note for _text_ we should add it for _literal_ too?

Reply:
To be fair, I don't really see the benefit of all this duplication in syntax.md.
The abnf should be enough.
It only makes it difficult to keep them in sync.
I just found that `syntax.md` contains an exact copy of the abnf section I am changing.

Probably too late to refactor this though...

Reply:
I added the same note to the quoted literals.

And changed in a couple of places this is affects quoted literals, not all literals.
(see graph in the main comments section)